release.config.js

```diff
@@ -1,16 +1,82 @@
-const { DEPLOY } = process.env;
+const { getPackagePath, getBuildPath } = require('./webpack/packageNaming');
 
-const getLogger = require('semantic-release/lib/get-logger');
+const release = Object.freeze({
+  verifyConditions: [
+    '@semantic-release/changelog',
+    '@semantic-release/git',
+    '@semantic-release/github'
+  ],
+  analyzeCommits: {
+    preset: 'angular',
+    parserOpts: {
+      noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'MAJOR RELEASE']
+    }
+  },
+  prepare: [
+    '@semantic-release/changelog',
+    {
+      path: '@semantic-release/exec',
+      cmd:
+        'sed -i -r \'s/"version":\\s*"[^"]+"/"version": "${nextRelease.version}"/\' package.json'
+    },
+    {
+      path: '@semantic-release/git',
+      assets: ['package.json', 'yarn.lock', 'CHANGELOG.md'],
+      message: 'chore: release ${nextRelease.version}\n\n${nextRelease.notes}'
+    }
+  ],
+  publish: [
+    {
+      path: '@semantic-release/exec',
+      cmd: 'yarn run buildVersion'
+    },
+    {
+      path: '@semantic-release/exec',
+      cmd: 'yarn run upload:firefox:staging'
```

Comment:
I might miss something here, can we still "release" *staging* and *proding* version without publishing *production* version?